Makefile

```diff
@@ -2,10 +2,25 @@
 # Tools                                 #
 #########################################
 
+VERSION             ?= $(shell cat VERSION)
+REGISTRY            ?= europe-docker.pkg.dev/gardener-project/snapshots
+IMAGE_REPOSITORY    := $(REGISTRY)/gardener/etcd-wrapper
+IMAGE_TAG           := $(VERSION)
+IMG ?= ${IMAGE_REPOSITORY}:${IMAGE_TAG}
+
 BIN_DIR := bin
 
 include hack/tools.mk
 
+.PHONY: docker-build
+docker-build:
```

Comment:
can you for now remove these targets as they are very g/g specific. Later we will add scripts to build and push docker images which will take in any registry as argument.

Reply:
done. PTAL